Agents/PropertyValueEstimationAgent/tests/mockutils/stack_configs_mock.py

```diff
@@ -0,0 +1,17 @@
+# NOTE the endpoints provided here are suitable for running the agent tests both locally and as dockerised tests 
+# In the latter case `localhost` gets replaced with `docker.host.internal` while building the Docker image to ensure
```

Comment:
Please change to `host.docker.internal` in the comment, instead of `docker.host.internal`.

Reply:
Addressed by [70bf83d](https://github.com/cambridge-cares/TheWorldAvatar/pull/454/commits/70bf83da2a3c88d28f664d112439d09e18003cba)